### PR TITLE
Tested on a newer version of macOS Big Sur

### PR DESCRIPTION
**Pull request by crusher083**

### CODE REVIEW

**Comment by thomasaarholt** [comment]
Thanks! :) I'll let the tests run, just as a formality, and then I'll merge!

**Comment by thomasaarholt** [comment]
@ericpre could you take a look at why the ubuntu tests are failing?

File: README.rst [hunk 1]
```diff
@@ -96,7 +96,7 @@ cog will be visible in the menu bar when the processes are running. Once you
 have finished with the server then manually kill the process via the
 drop-down menu from this spinning cog.
 
-The launchers have been tested on macOS Mojave (10.14.6).
+The launchers have been tested on macOS Big Sur (11.5.1).
```
Comment (ericpre) [inline README.rst:99]:
```suggestion
The launchers have been tested on macOS Mojave (10.14.6), Catalina (10.15) and Big Sur (11.1-11.5.1).
```

**Review by ericpre — COMMENTED** [review]
Thanks! It would be better to keep what version are supported instead of overwriting it. I have got confirmation that it has been working fine from 10.14.6 to latest, so I suggested to change to a list of versions.

**Comment by ericpre** [comment]
> @ericpre could you take a look at why the ubuntu tests are failing?

```
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/w/webkit2gtk/libjavascriptcoregtk-4.0-18_2.32.0-0ubuntu0.20.04.1_amd64.deb  404  Not Found [IP: 52.252.75.106 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/w/webkit2gtk/libwebkit2gtk-4.0-37_2.32.0-0ubuntu0.20.04.1_amd64.deb  404  Not Found [IP: 52.252.75.106 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```
It seems that some packages can't be installed on ubunut.

**Comment by ericpre** [comment]
Thanks @crusher083!

**Comment by ericpre** [comment]
The failure was transient: the [merge commit](https://github.com/hyperspy/start_jupyter_cm/commit/b2733f62603e809280ef1da8e7c03ed680d37b9d) doesn't fail.

**Pull request merged**